File: openapi-generator/csharp-dotnet2/src/main/CsharpDotNet2/Org/OpenAPITools/Model/Machine.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// Contains information about a machine.
  /// </summary>
  [DataContract]
  public class Machine {
    /// <summary>
    /// ID of the machine.
    /// </summary>
    /// <value>ID of the machine.</value>
    [DataMember(Name="id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "id")]
    public long? Id { get; set; }

    /// <summary>
    /// Name of the machine.
    /// </summary>
    /// <value>Name of the machine.</value>
    [DataMember(Name="name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "name")]
    public string Name { get; set; }

    /// <summary>
    /// Notes about the machine
    /// </summary>
    /// <value>Notes about the machine</value>
    [DataMember(Name="notes", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "notes")]
    public string Notes { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Machine {\n");
      sb.Append("  Id: ").Append(Id).Append("\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Notes: ").Append(Notes).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: openapi-generator/dart/lib/model/machine.dart
part of openapi.api;

class Machine {
  /* ID of the machine. */
  int id = null;
  /* Name of the machine. */
  String name = null;
  /* Notes about the machine */
  String notes = null;
  Machine();

  @override
  String toString() {
    return 'Machine[id=$id, name=$name, notes=$notes, ]';
  }

  Machine.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['id'] == null) {
      id = null;
    } else {
          id = json['id'];
    }
    if (json['name'] == null) {
      name = null;
    } else {
          name = json['name'];
    }
    if (json['notes'] == null) {
      notes = null;
    } else {
          notes = json['notes'];
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'notes': notes
    };
  }

  static List<Machine> listFromJson(List<dynamic> json) {
    return json == null ? new List<Machine>() : json.map((value) => new Machine.fromJson(value)).toList();
  }

  static Map<String, Machine> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, Machine>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new Machine.fromJson(value));
    }
    return map;
  }
}

